asv_benchmarks/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,107 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the airspeed velocity docs for more information.
+# https://asv.readthedocs.io/en/stable/
+import numpy as np
+from numpy import newaxis
+from scipy.stats import circstd
+from scipy.sparse import coo_matrix
+import scipy.signal as ss
+import warnings
+import arviz as az
+from arviz.stats.stats_utils import _circular_standard_deviation, histogram, stats_variance_2d
+from arviz.kde_utils import _fast_kde, _fast_kde_2d
+
+
+class Hist:
+    params = (True, False)
+    param_names = "Numba"
+
+    def setup(self, numba_flag):
+        self.data = np.random.rand(10000, 1000)
+        if numba_flag:
+            az.Numba.enable_numba()
+        else:
+            az.Numba.disable_numba()
+
+    def time_histogram(self, numba_flag):
+        histogram(self.data, bins=100)
+
+
+class Variance:
```

Comment:
variance benchmark is failing, could you look into it? https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=2274&view=logs&j=273a7ac9-cc40-5d2f-3776-d61d22a0ef9c&t=8fe29edb-6726-5e00-142e-7c337980bc77&l=99

Reply:
I was trying to debug this so I ran the benchmarks with `--show-stderr` flag but it only showed
```
asv: benchmark timed out (timeout 60.0s)
```
And when I ran the benchmarks with both `--quick --show-stderr` flags, it worked but the function took 31s to run with `Numba=False` and when I used the function in a notebook with the same data and numba disabled it ran in milliseconds. I think it's some issue with the benchmarks configuration, I'll continue looking.

Reply:
My bad, the data was too large so the function was actually really slow without Numba speed-up  (taking ~30s and not milliseconds after I rechecked) and it caused the benchmark to fail :(

I'll reduce the data.